tgui/packages/tgui/interfaces/WizardApprenticeContract.js

```diff
@@ -0,0 +1,66 @@
+import { useBackend } from '../backend';
+import { Button, Box, LabeledList, Section, Divider, Fragment } from '../components';
+import { Window } from '../layouts';
+
+export const WizardApprenticeContract = (props, context) => {
+  const { act, data } = useBackend(context);
+  // Extract `health` and `color` variables from the `data` object.
+  const {
+    used,
+  } = data;
+  let body;
+  if (used) {
+    body = (
+      <Box bold color="bad">
+        You have already summoned your apprentice.
```

Comment:
Questioning whether its worth to just show this as a chat message instead of an entire UI section which does nothing.